### ISSUE TITLE
How should I adjust the hyperparameters to train a strong DDPM model based on CelebA?

### ISSUE THREAD

**Issue by LinWeiJeff** [Tobi-r9/RaMViD#14]
I'm very grateful for your work about this github repository!!! However, I have some questions about the hyperparameters to want to ask you.
I know your repository is based on [improved-diffusion](https://github.com/openai/improved-diffusion), and I want to use [improved-diffusion](https://github.com/openai/improved-diffusion) to train a strong DDPM model based on my CelebA dataset consisting of 202,599 align&cropped human face images (each image is 218(height)x178(width) pixels) for human face image synthesis task and then use the saved model to perform human face image inpainting task.

**Could you give me some suggestions about how to adjust the hyperparameters used in [improved-diffusion](https://github.com/openai/improved-diffusion) to train a DDPM model as strong as models provided by [improved-diffusion](https://github.com/openai/improved-diffusion) for human face image synthesis and then inpainting task?**
By the way, for dataset, should I need to resize the images in my dataset to 256x256 firstly?

I hope the trained strong DDPM model learns the features of human face well so it can be used for human face image synthesis task and human face image inpainting task (i.e. recovering the masked parts of a masked human face image).

I want to know how to adjust the values of the hyperparameters in **diffusion_defaults() of script_util.py**, **model_and_diffusion_defaults() of script_util.py** and **create_argparser() of image_train.py** in [improved-diffusion](https://github.com/openai/improved-diffusion) to train a strong denoising model in DDPM based on my CelebA dataset.

I have tried some combinations of the hyperparameters used in [improved-diffusion](https://github.com/openai/improved-diffusion) to train, however, the human face image inpainting results of the saved model files `ema_0.9999_XXX.pt` and `modelXXX.pt` are both bad.

In addition, because of the limitation of my GPU memory, I set the value of the hyperparameter `num_channels` only 64, I want to know if this hyperparameter affects the performance of the traind DDPM model. Should I try to set it larger?

In conclusion, I hope you can give me some suggestions about how to adjust the hyperparameters used in [improved-diffusion](https://github.com/openai/improved-diffusion) to let me train a strong DDPM model for human face image synthesis and inpainting tasks.

Thanks a lot for your any help!!!
p.s. I directly and manually set up the values of the hyperparameters in the codes of [improved-diffusion](https://github.com/openai/improved-diffusion) not through any flag.
